CHANGELOG.md

```diff
@@ -1,48 +1,62 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## 1.4.0 - 2024-06-25
-[GRD-797](https://jira.oicr.on.ca/browse/GRD-797) - add vidarr labels to outputs (changes to medata only)
+## [1.4.0] - 2024-06-25
+### Added
+- [GRD-797](https://jira.oicr.on.ca/browse/GRD-797) - Add vidarr labels to outputs (changes to medata only)
+
+## [1.3.1] - 2023-09-16
+### Changed
+- Modified gridss module (reduced time spent generating metrics plots). 
 
-## 1.3.1 - 2023-09-16
-- modified gridss module (reduced time spent generating metrics plots). Added optional parameter to gridss tasks (additionalParameters)
+### Added
+- Added optional parameter to gridss tasks (additionalParameters).
 
-## 1.3.0 - 2023-08-01
-- svprep job scattered by chromosome, RAM allocated based on chromosome size and additional performance-affecting
-  parameters are exposed for tuning
+## [1.3.0] - 2023-08-01
+### Changed
+- svprep job scattered by chromosome, RAM allocated based on chromosome size and additional performance-affecting parameters are exposed for tuning
 
-## 1.2.0 - 2023-05-24
-- Added step for generating names for tumor and normal, basically the same way mutect2 uses
+## [1.2.0] - 2023-05-24
+### Added
+- Added step for generating names for tumor and normal, basically the same way mutect2 uses.
 
-## 1.1.0 - 2023-04-08
-- Added additional pre-processing step
+## [1.1.0] - 2023-04-08
+### Added
+- Added additional pre-processing step.
 
-## 1.0.1 - 2023-01-09
-- Added new gridss aliases (gridss_matched and gridss_matched_by_tumor_group)
+## [1.0.1] - 2023-01-09
+### Added
+- Added new gridss aliases (gridss_matched and gridss_matched_by_tumor_group).
 
-## 1.0.0 - 2022-12-13
-- modified README.md
-- added parameter blacklist
-- removed tumor/normal name parameters, added outputFilePrefix
-- modified RT 
+## [1.0.0] - 2022-12-13
+### Changed
+- Modified README.md
+- Modified RT 
 - Split the workflow in parts to improve speed/reliability
 
-## Unreleased
+### Added
+- Added parameter blacklist
+
+### Removed
+- Removed tumor/normal name parameters, added outputFilePrefix
+
+## [Unreleased]
 
-## 0.0.1 - 2022-10-27
+## [0.0.1] - 2022-10-27
 ### Added
 - CHANGELOG.md
 - vidarrbuild.json
 - vidarrtest-regression.json
 - Jenkinsfile
 
-## 0.0.1 - 2022-10-31
+## [0.0.1] - 2022-10-31
```

Comment:
This should probably say 'Unreleased' - otherwise it looks like a duplication of a tag